src/lib_json/json_value.cpp

```diff
@@ -1401,7 +1401,7 @@ void Value::Comments::set(CommentPlacement slot, String comment) {
   if (!ptr_) {
     ptr_ = std::unique_ptr<Array>(new Array());
   }
-  // check comments array boundry.
+
```

Comment:
I don't really want the blank line either.
But while we're manipulating this function let's make the change more meaningful.

```c++
void Value::Comments::set(CommentPlacement slot, String comment) {
  if (slot >= CommentPlacement::numberOfCommentPlacement)
    return;
  if (!ptr_)
    ptr_ = std::unique_ptr<Array>(new Array());
  (*ptr_)[slot] = std::move(comment);
}
```

In other words, if we're not actually going to write to `ptr_`, we don't need to make one. We don't need to allocate it until we actually use it, and errant calls to `set` will then have no effect, which is a cleaner recovery I think.

Reply:
I agree with your opinion.